references/validationFunctions/specimen/stageGroup.js

```diff
@@ -28,7 +28,12 @@ const validation = () =>
   (function validate(inputs) {
     const {$row, $name, $field} = inputs;
     let result = { valid: true, message: 'Ok' };
-    if ($row.pathological_tumour_staging_system && $field) {
+    if (!($row.pathological_tumour_staging_system) && $field) {
+       result.valid = false;
+       const msg = "'pathological_tumour_staging_system' must be submitted if 'pathological_stage_group' is submitted";
```

Comment:
`The field` 'pathological_tumour_staging_system' must be submitted if 'pathological_stage_group' is submitted"`.`